scripts/setup_data_file.py

```diff
@@ -5,12 +5,14 @@
 
 This script has no command line options.
 """
-
+from src.data.scout_census import ScoutCensus
 from src.data.scout_data import ScoutData
 from src.data.ons_pd_may_19 import ONSPostcodeDirectoryMay19
 
 if __name__ == "__main__":
-    scout_data = ScoutData(csv_has_ons_pd_data=False)
-    ons_pd = ONSPostcodeDirectoryMay19(scout_data.settings["ONS PD location"])
+    scout_data = ScoutData(merged_csv=False)
+    for section in ScoutCensus.column_labels["sections"].keys():
+        scout_data.data[f"{section}_total"] = scout_data.data[[f"{section}_f", f"{section}_m", f"{section}_SelfIdentify", f"{section}_PreferNoToSay"]].sum(axis=1).astype("Int16")
```

Comment:
I like it - so succinct and pretty beautiful. This is only a suggestion, so take it or leave it ... I quite like having all the 'magic' strings in one place, so I wonder if the list: `["Beavers_f", ... ] should be a value in the 'sections dictionary'.